src/digraph.rs

```diff
@@ -317,10 +320,10 @@ impl PyDiGraph {
     fn __getnewargs_ex__<'py>(&self, py: Python<'py>) -> (Py<PyTuple>, Bound<'py, PyDict>) {
         (
             (self.check_cycle, self.multigraph, self.attrs.clone_ref(py)).into_py(py),
-            [
-                ("initial_node_count", self.graph.node_bound()),
-                ("initial_edge_count", self.graph.edge_bound()),
-            ]
+            indexmap::indexmap! {
+                "node_count_hint" => self.graph.node_bound(),
+                "edge_count_hint" => self.graph.edge_bound(),
+            }
```

Comment:
Do we need to make the intermediate `IndexMap` here? This is basically doing two allocations now, one for the indexmap and one for the pydict while before it was just a single one for the pydict.

Reply:
I reverted the change, I just really wish there was a macro to make Python dictionaries. Creating a list of tuples doesn't tell the reader that it's a dictionary in my opinion.